incident_viewer_demo/incident/incident.go

```diff
@@ -40,52 +44,29 @@ type DefaultIncident struct {
 	badEndTime       time.Time
 	avgGoodDS        float64
 	avgBadDS         float64
+	asn              string
+	location         string
 	severity         float64
 	numTestsAffected int
 	goodPeriodInfo   string
 	badPeriodInfo    string
 	incidentInfo     string
 }
 
-/* Create an IncidentData object to be stored in JSON format */
-func (i *IncidentData) MakeJsonIncident(
-	goodTimeStart time.Time,
-	goodTimeEnd time.Time,
-	badTimeStart time.Time,
-	badTimeEnd time.Time,
-	goodMetric float64,
-	badMetric float64,
-	severity float64,
-	testsAffected int,
-	goodPeriodInfo string,
-	badPeriodInfo string,
-	incidentInfo string) {
-
-	i.GoodPeriodStart = goodTimeStart
-	i.GoodPeriodEnd = goodTimeEnd
-	i.BadPeriodStart = badTimeStart
-	i.BadPeriodEnd = badTimeEnd
-	i.GoodPeriodMetric = goodMetric
-	i.BadPeriodMetric = badMetric
-	i.Severity = severity
-	i.NumTestsAffected = testsAffected
-	i.GoodPeriodInfo = goodPeriodInfo
-	i.BadPeriodInfo = badPeriodInfo
-	i.IncidentInfo = incidentInfo
-}
-
 /* Assign data members and set appropriate text for information fields */
 func (i *DefaultIncident) MakeIncidentData(goodStartTime time.Time, goodEndTime time.Time,
 	badStartTime time.Time,
 	badEndTime time.Time,
 	avgGoodDS float64,
 	avgBadDS float64,
+	asn string,
+	location string,
 	severity float64,
 	testsAffected int) {
 
 	// Convert strings to be float64 types described by 2 decimal places
 	gds := strconv.FormatFloat(avgGoodDS, 'f', 2, 64)
```

Comment:
what is 'f' character? Is there a way you can make what the 'f' does more intuitive so that one does not have to read the documentation of FormatFloat

Reply:
I added a comment. I think that people should be encouraged to look at the original documentation if they wanted to change it since there are some complicated nuances in the different arguments.